From 641dacf8a5e51d81d32cd72c3d312bf1f06bd9bf Mon Sep 17 00:00:00 2001
From: Lucio Delelis <ldelelis@est.frba.utn.edu.ar>
Date: Sat, 12 May 2018 00:56:47 -0300
Subject: [PATCH] Replaced some calls to the removed succeeded attribute

According to documentation, the correct way is to consult 'ok' now
---
 patchwork/files.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/patchwork/files.py b/patchwork/files.py
index 081c0ee..9d0f1af 100644
--- a/patchwork/files.py
+++ b/patchwork/files.py
@@ -28,7 +28,7 @@ def exists(c, runner, path):
     Return True if given path exists on the current remote host.
     """
     cmd = 'test -e "$(echo {})"'.format(path)
-    return runner(cmd, hide=True, warn=True).succeeded
+    return runner(cmd, hide=True, warn=True).ok


 @set_runner
@@ -54,7 +54,7 @@ def contains(c, runner, filename, text, exact=False, escape=True):
         if exact:
             text = "^{}$".format(text)
     egrep_cmd = 'egrep "{}" "{}"'.format(text, filename)
-    return runner(egrep_cmd, hide=True, warn=True).succeeded
+    return runner(egrep_cmd, hide=True, warn=True).ok


 @set_runner
